### PR TITLE
feat(optimize): add optimize c8 userTask docs

### PR DESCRIPTION
**Pull request by PHWaechtler**
related to [OPT#22345](https://github.com/camunda/camunda-optimize/issues/11245)

## Description

Rewrites and updates the documentation for userTask reporting in Optimize for 3.13.0 release, see content [here](https://docs.google.com/document/d/12cSV__xVt9ipG2nGAhD6ADzjiDwPrrj4d7zgd_IJ7cQ/edit).

## When should this change go live?

- [x] This change is not yet live and should not be merged until April 9th (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by christinaausley — COMMENTED** [review]
Standard review in latest commit, but I largely just want to ensure we either link to, or clarify the differentiation between references of "filters" as documented in https://docs.camunda.io/optimize/components/userguide/decision-analysis/decision-filter/.

**Comment by christinaausley** [comment]
As an FYI, I think we may also need to add this to the sidebar 👍

File: optimize/self-managed/optimize-deployment/configuration/system-configuration-platform-8.md [hunk 1]
```diff
@@ -17,11 +17,22 @@ description: "Connection to Camunda 8."
...
+to the [multi-tenancy documentation](./multi-tenancy.md).
 
 To use multi-tenancy, the feature must be enabled across all components.
 
 | YAML path                  | Default value | Description                                              |
 | -------------------------- | ------------- | -------------------------------------------------------- |
 | multitenancy.enabled       | false         | Enables the Camunda 8 multi-tenancy feature in Optimize. |
 | security.auth.ccsm.baseUrl | null          | The base URL of Identity.                                |
+
+### Settings related to Camunda 8 native UserTasks
```
Comment (toco-cam) [inline optimize/self-managed/optimize-deployment/configuration/system-configuration-platform-8.md:30]: @PHWaechtler isn't that "Zebee user task" (Term used by tasklist)

File: optimize/components/userguide/process-analysis/user-task-analytics.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+To maintain adherence to Service Level Agreements (SLAs), it is essential to establish clear and measurable targets for each user task within a process. These targets serve as performance benchmarks that guide users and help maintain the quality and timeliness of task completion. Optimize allows users to define target values for user tasks on the heatmap.
+
+![user task target example](./img/userTask_target_heatMap.png)
+
+### Evaluate task performance
+
+For a thorough understanding of task execution within your processes, it is imperative to examine how tasks are performed both by the nature of the user task itself and by the assignee responsible for completing the task. This detailed analysis enables organizations to assess individual and collective task performance, providing insights into workflow efficiency and productivity. Optimize allows breaking down user tasks by user task or assignee for the assigned and unassigned state of the task lifecycle. With filters, it is possible to evaluate only a subset of the process instances, for example, to evaluate the performance by country.
+
+![user filtered report example](./img/userTask_report_filters.png)
```
Comment (toco-cam) [inline optimize/components/userguide/process-analysis/user-task-analytics.md:38]: @PHWaechtler where to find the "Features" and "Good to know" section?

File: optimize/self-managed/optimize-deployment/configuration/system-configuration-platform-8.md (hunk 1 above)
Comment (PHWaechtler) [reply optimize/self-managed/optimize-deployment/configuration/system-configuration-platform-8.md:30]: I dont know, this is the term you told me will be used when we initially discussed the docs. Should it be changed to `Zeebe user task`?

**Comment by PHWaechtler** [comment]
> As an FYI, I think we may also need to add this to the sidebar 👍

Thanks @christinaausley - is this something I should add to the PR?

**Comment by christinaausley** [comment]
> > As an FYI, I think we may also need to add this to the sidebar 👍
> 
> Thanks @christinaausley - is this something I should add to the PR?

Yes please 😄 In `optimize_sidebars.js` and `sidebars.js` -- let me know if I can help!

**Review by toco-cam — APPROVED** [review]

**Comment by christinaausley** [comment]
@PHWaechtler [we cut the release yesterday](https://camunda.slack.com/archives/C026U8GBNSW/p1712630002594749?thread_ts=1712628310.130439&cid=C026U8GBNSW), so may need to backport to `8.5` here. Let me know if you would like assistance 👍

**Comment by PHWaechtler** [comment]
> @PHWaechtler [we cut the release yesterday](https://camunda.slack.com/archives/C026U8GBNSW/p1712630002594749?thread_ts=1712628310.130439&cid=C026U8GBNSW), so may need to backport to `8.5` here. Let me know if you would like assistance 👍

@christinaausley I've backported and rebased as well, let me know if anything else needs added :)

**Review by christinaausley — APPROVED** [review]
Thanks for pushing through this one @PHWaechtler!

**Review by toco-cam — COMMENTED** [review]
Ok

**Pull request merged**